docker-compose.yml

```diff
@@ -19,6 +21,7 @@ services:
     env_file:
       - './api/.env'
   frontend:
+    container_name: 'tapp_fe'
```

Comment:
The frontend container should only run in the dev compose file because the container is an instance of webpack-dev-server.